includes/actions/HistoryAction.php

```diff
@@ -432,7 +436,7 @@ function doBatchLookups() {
 		$this->mResult->seek( 0 );
 		$batch = new LinkBatch();
 		foreach ( $this->mResult as $row ) {
-			if( !empty( $row->user_name ) ) {
+			if( !isset( $row->user_name ) ) {
```

Comment:
I believe it should be `isset` - without the negation.

Reply:
Aye, goes without sayin…